marketplace_builder/views/pages/tutorials/blog/installing-blog-module.liquid

```diff
@@ -12,51 +12,66 @@ This guide will help you install the PlatformOS Blog Module.
+Log in to the [Partner Portal](https://portal.apps.near-me.com/) and go to your Instance detail view. 
+At the bottom of the page in the Available modules section search for "PlatformOS Blog" and click Install on the right. You will receive an email notification when the installation process is complete.
 
-<pre class="command-line" data-user="user" data-host="host"><code class="language-bash">
-npm install --no-save @platform-os/blog
-</code></pre>
+It might be necesary to reindex your instance after blog installation so go to your instance detail view in Partner Portal and follow up with "Rake Tasks" at the top navigation. Click run next to `elasticsearch:create_index`.
 
-The installer starts and asks you the following:
+### Step 2: Create Blog Administrator
 
-* `? Your blog path:`: Enter the path where you would like to display your blog, for example `blog`. Don't enter the full URL.
-* `? Your blog type is:`: Select the type of your blog with the up and down arrow keys. The **Instance blog** is the main blog of the site, where the Instance owner can create and edit posts. **User blogs** allow any user to create a blog, and the owner of the blog can create and edit posts.
-* `? Please provide name of user profile type that is allowed to access admin part of the blog:`: Every user has an `user_profile_type` (user type, e.g. default or admin). Depending on what your site does (e.g. connects experts with lawyers) you can have additional user profile types (e.g. expert and lawyer). Specify here which users will be able to access the admin interface of the blog by entering their user profile type. Enter `default` to use the default user profile type.
-* `? Your blog layout name:`: You can have a layout specific to your blog (e.g. widgets, columns, etc.), specify it here. To use the default blog layout provided by the Blog Module enter `blog-layout`.
+A simple administration panel is included in the Blog module, where you can configure your blog and create its content. By default, this part is restricted for registered users with Administrator role. 
 
-The installer runs and the Blog Module is installed.
+In order to create your first Admin:
 
-{% include 'alert/note', content: 'In this example, you used the default user profile, so any user can edit posts on the Instance blog. To prevent unauthorized use, it is recommended to create an admin user profile type and adjust the blogs <a href="/tutorials/authorization-policy/authorization-policy">Authorization Policies</a>.' %}
+1. Visit your instance page.
+2. Go to the signup page by appending `/sign-up` to your instance address in your browser's address bar. 
+3. Fill in the form and create your first user. 
+4. You will be redirected to the `/log-in` page, where you need to provide the same user login and password. 
+5. Blog comes with a handy migration which grants Administrator role to all existing users:
 
-### Step 2: Deploy blog
+    ```bash
```

Comment:
too much indentation - i dont know what will happen to this, but it might break things

Reply:
I checked it on my staging, it seems to be fine.